### PR TITLE
add a home page css

### PR DESCRIPTION
**Pull request by kaysar314**
. modified setting file, able to access static file

. create home page css and modified somewhere in base.html and index.html

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,104 @@
+led / optimized / DLL files
```
Comment (sptzxbbb) [inline .gitignore]: Try to filter unwanted file with regex and keep it clean.

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+# virtualenv
+venv/
+ENV/
+
+# Spyder project settings
+.spyderproject
+
+# Rope project settings
+.ropeproject
```
Comment (sptzxbbb) [inline .gitignore]: Try to filter unwanted files with regex and keep it clean.

Comment (kaysar314) [inline .gitignore]: okay

**Comment by sptzxbbb** [comment]
Please code in dev branch and be cautious about master.

**Pull request closed without merging**